optimum/intel/openvino/configuration.py

```diff
@@ -67,7 +67,15 @@ class OVQuantizationMethod(str, Enum):
         "quant_method": OVQuantizationMethod.AWQ,
     },
     "meta-llama/Llama-2-7b-hf": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.6},
-    "meta-llama/Llama-2-7b-chat-hf": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.8},
+    "meta-llama/Llama-2-7b-chat-hf": {
+        "bits": 4,
+        "sym": True,
+        "group_size": 128,
+        "ratio": 0.8,
```

Comment:
Did you check the Similarity with ratio=1 with all other remaining parameters? If it's in our threshold, I would use the default ratio, i.e. 1

Reply:
Updated to 1.0. Surprisingly, similarity increased by another 2%.